web/app/main/config.js

```diff
@@ -2,8 +2,17 @@
 // -- Application Configuration
 
 config = {
-    context : '../api/json',
+    // Run with file system access has problem with new browsers
+    // context : '../api/json',  
+    // useApiMethod : 'get',  // but not working with new browsers
+    
+    // Run with HTTP web server, content in wwwroot/KickStart directory
+    context : 'http://localhost/KickStart/api/json',  
```

Comment:
It is better to user relative urls, not absolute, and starting with "http://..." makes it absolute. If it is relative, you can run it from any folder :)